src/components/app-content-renderer/SimpleSlider.js

```diff
@@ -59,30 +58,27 @@ function SimpleSlider() {
                 </Text>
                 <Text component="p" className="pf-u-flex-grow-1">
                   Try Red Hat OpenShift technologies without setup or
-                  configuration for free with your Red Hat Developer account.
-                </Text>
-                <Flex className="pf-u-flex-direction-row-on-md">
-                  <Text component="p" className="pf-u-pr-sm pf-u-mb-0">
-                    <Link to="/openshift/sandbox">
-                      <Button variant="danger" className="pf-u-pr-sm">
-                        Build apps on the sandbox
-                      </Button>
-                    </Link>
+                  configuration for free with your Red Hat Developer
+                  account.&nbsp; &nbsp;
+                  <Text
+                    component="a"
+                    className="pf-u-active-color-400"
+                    href="https://developers.redhat.com/about"
+                  >
+                    Learn about the program.
                   </Text>
-                  <Text component="a">
-                    <Button
-                      href="https://developers.redhat.com/about"
-                      variant="tertiary"
-                      className="land-c-button-custom"
-                    >
-                      Learn about the program
+                </Text>
+                <Text component="p" className="pf-u-pr-sm pf-u-mb-0">
+                  <Link to="/openshift/sandbox">
```

Comment:
This doesn't follow the `beta`/`preview` either use `openshift/sandbox` (which will just append the text) or `./openshift/sandbox` which will use baseURI parameter. Or import `useChrome` and call `appNavClick`

Reply:
@karelhala Every link on the Landing page is set up that same way. Do they all need to be changed? If so, can that be done separately?

Reply:
Yeah, might be better to go trough the routes and fix them properly to react-router v6.